pycompwa/data/_labels.py

```diff
@@ -0,0 +1,10 @@
+"""Hidden globals within the :mod:`pycompwa.data` module.
+
+These labels are used to consistently name the columns of the
+:class:`pandas.DataFrame`.
+"""
+
+
+ENERGY = 'E'
+MOMENTA = ['p_x', 'p_y', 'p_z', ENERGY]
+WEIGHT = 'weight'
```

Comment:
Maybe its better to put this with the pandas accessor, since this defines the these labels anyway. Also here you could use an `Enum`.
```
class _Labels(Enum):
    ENERGY = 'E'
    MOMENTA = ['px', 'py', 'pz', ENERGY]
    WEIGHT = 'weight'
```

Reply:
True, I tried that in the beginning, but the labels are also used in other submodules (`create`, `convert`, `io`). So if you include the labels in the data module itself, you get cyclic dependencies.

Reply:
Hmm ok. Maybe its better to keep `__init__.py` clean then and move the accessor to some other module. I think this could also be merged with the `create` module. This is not crucial for pycompwa, but when things will be ported to tensorwaves it should be checked in detail! ComPWA/tensorwaves#23